README.md

```diff
@@ -57,6 +61,12 @@ Start by installing the package:
 pip install hayhooks
 ```
 
+If you want to use the [MCP server](#mcp-server), you need to install the `hayhooks[mcp]` package:
```

Comment:
should we mention that Python 3.10 is needed in this case?

Reply:
Added two notes, thanks!